.idea/codeStyles/Project.xml

```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
We don't want any change in `.idea` folder, please remove those files from your commit. (This does not mean that you delete them, it simply means that those files will not be committed when making a PR.)

Reply:
@gunishjain We don't need these changes in these files.